enforce encoding='utf8'

haluktutuk opened the pull request:
Installation on Windows 10 is not possible unless we enforce encoding='utf8'

haluktutuk committed:
Update setup.py

enforce encoding='utf8'

Signed-off-by: Haluk Tutuk <56925617+haluktutuk@users.noreply.github.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/StarRocks/dbt-starrocks?pullRequest=39) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/StarRocks/dbt-starrocks?pullRequest=39) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/StarRocks/dbt-starrocks?pullRequest=39) it.</sub>

haluktutuk committed:
Update connections.py

fixed `No module named 'json.decoder.json'; 'json.decoder' is not a package` error

Signed-off-by: Haluk Tutuk <56925617+haluktutuk@users.noreply.github.com>

haluktutuk committed:
Update setup.py

upgraded the dependency versions

Signed-off-by: Haluk Tutuk <56925617+haluktutuk@users.noreply.github.com>

haluktutuk committed:
Update dev_requirements.txt

updated

Signed-off-by: Haluk Tutuk <56925617+haluktutuk@users.noreply.github.com>

Commit:
upgraded version

Commit:
upgraded

alberttwong commented:
@haluktutuk can you sign the DCO and CLA?

Astralidea commented:
This problem should be solved, you can try to use 1.7.0
